stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/event/EventBatchUpdate.kt

```diff
@@ -42,12 +44,17 @@ internal class EventBatchUpdate private constructor(
         getCurrentChannel(cid)?.also { channel ->
             channel.updateLastMessage(message)
 
-            val currentUserId = domainImpl.user.value?.id
+            val currentUserId = GlobalMutableState.getOrCreate().user.value?.id ?: return
```

Comment:
Early return to improve readability since we don't do anything inside the function body if this is null.